charts/plane-enterprise/templates/workloads/email.deployment.yaml

```
@@ -0,0 +1,94 @@
+  annotations:
+    reloader.stakater.com/auto: "true"
+spec:
+  replicas: {{ .Values.services.email_service.replicas | default 1 }}
+  selector:
+    matchLabels:
+      app.name: {{ .Release.Namespace }}-{{ .Release.Name }}-email-app
+  template:
+    metadata:
+      namespace: {{ .Release.Namespace }}
+      labels:
+       app.name: {{ .Release.Namespace }}-{{ .Release.Name }}-email-app 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)